lib/structures/Role.js

```diff
@@ -47,6 +50,21 @@ class Role extends Base {
         if(data.permissions !== undefined) {
             this.permissions = new Permission(data.permissions_new || data.permissions);
         }
+        if(data.tags !== undefined) {
+            if(data.tags.bot_id !== undefined) {
+                this.botID = data.tags.bot_id;
+            }
+            if(data.tags.integration_id !== undefined) {
+                this.integrationID = data.tags.integration_id;
+            }
+            if(data.tags.premium_subscriber !== undefined) {
+                this.premiumSubscriber = true;
```

Comment:
any particular reason for making this a top level field instead of being consistent with discord here?